core/src/main/webapp/workers.jsp

```diff
@@ -30,9 +30,9 @@
                 <% } %>
                 <th>Last Heartbeat</th>
                 <th>State</th>
-                <th>Memory Capacity</th>
-                <th>Used Memory</th>
-                <th>Memory Usage</th>
+                <th>Workers Capacity</th>
+                <th>Space Used</th>
+                <th>Space Usage</th>
```

Comment:
Are we going to show every worker's hierarchy store info (i.e. mem usge, hdd usge) on this "workers" page as well, instead of only showing the total capacity usage? It would be nice to know these info and we do have these data in master now.